syatt-io/b2b-organizations
# Feature/witi 148 auto approve org req

#### What problem is this solving?

Admin for b2b settings to have auto approval of new organizations possible

#### How to test it?

You will need the b2b organization graphql repo to link as well for this to work properly

Go to [https://witi204--whitebird.myvtex.com/admin/b2b-organizations/organizations/#/settings](https://witi204--whitebird.myvtex.com/admin/b2b-organizations/organizations/#/settings) and the fields should be populated with the settings from masterdata

If auto approval is set to yes, go to [https://witi204--whitebird.myvtex.com/account#/organization](https://witi204--whitebird.myvtex.com/account#/organization) with an account without an organization and sign up. It should auto approval the organization for you

#### Screenshots or example usage:
![Screen Shot 2022-07-22 at 11 55 34 AM](https://user-images.githubusercontent.com/85588955/180477565-af3f125d-d483-4e1c-bf31-4d2ccbb40c7d.png)

#### Describe alternatives you've considered, if any.
I was thinking of instead of updating the request status to approved if auto approve is checked off, that we just create the organization instead. 

Let me know if that makes more sense

#### Related to / Depends on
https://github.com/syatt-io/b2b-organizations-graphql/pull/1
